ironfist/src/cpp/game/save.cpp

```diff
@@ -239,7 +239,7 @@ void SaveMapVariables(ironfist_map::map_t& m) {
 			char *s2 = "' could not be saved.";
 			int len = strlen(mapVariableId) + strlen(s1) + strlen(s2);
 			char *errorMessage = (char *)ALLOC(len);
-			snprintf(errorMessage, len, "%s%s%s", s1, mapVariableId, s2);
+			_snprintf_s(errorMessage, sizeof(errorMessage), len, "%s%s%s", s1, mapVariableId, s2);

```

Comment:
@Kert 

First, Github doesn't automatically notify people when you add commits to a pull request. In the future, please mention this in the Skype channel, or add a comment and tag me.

This is almost ready, but there's still a bug. sizeof(errorMessage) will actually be equal to 4, not len. "sizeof" is always a compile-time constant.

While we're at it, the calculation of "len" on line 240 is wrong -- there should be an extra +1 for the null terminator.